### ISSUE TITLE
Flash of incorrect style when loading a Zimit ZIM for first time with DarkReader or when using the random button

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-pwa#514]
While I have eliminated FOIS when using links within a loaded ZIM that has DarkReader enabled, incorrect style is still showing in the following circumstances:

* Loading a ZIM for the first time, or pressing the Home button
* Using the random button

These are minor issues, and it is a matter of porting the code that waits for DarkReader to have installed itself before showing the contents of the iframe, but it should be done for aesthetic reasons (and, marginally as an a11y issue).

**Comment by Jaifroid** [comment]
FOIS now affects most Zimit v2 ZIMs with DarkReader. The reason is because these ZIMs are faster to read (due to the pre-processing). It is difficult to detect when DarkReader has completed its transformations (this is some time after the `enabled` flag is set), so we have to introduce more time-based delay.
